cs464-group-project/src/pages/Players/PlayersMainPage.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import { getTeamsByLeagueName } from '../../components/Api/ApiRequest';
+import { Link } from 'react-router-dom';
+
+const apiKey = process.env.REACT_APP_API_KEY;
+
+async function getAllTeamNames() {
+  const leagueName = 'English Premier League';
+  try {
+    const data = await getTeamsByLeagueName(leagueName);
+    const teams = data.teams;
+
+    if (teams && teams.length > 0) {
+      const teamNames = teams.map((team) => team.strTeam);
+      console.log('Team Names:', teamNames);
+      return teamNames;
+    } else {
+      console.log('No teams found in the API response');
+      return [];
+    }
+  } catch (error) {
+    console.error('Error getting team names:', error.message);
+    throw error;
+  }
+}
```

Comment:
Seems like we're reusing the same function calls.  I can place this in the ApiRequest, that way we can just use something like getLeagueInfo(), instead of all this.

Reply:
Fixed and upated api calls